src/zenml/analytics/enums.py

```diff
@@ -85,6 +85,9 @@ class AnalyticsEvent(str, Enum):
     DEPLOY_STACK_COMPONENT = "Stack component deployed"
     DESTROY_STACK_COMPONENT = "Stack component destroyed"
 
+    # Cloud stack deployment
+    DEPLOY_STACK_CLOUD = "Stack deployed in cloud"
```

Comment:
Where else would a stack be deployed? 😅

Isn't there a better wording for this to explain what it really does?

Reply:
@schustmi there already is a DEPLOY_STACK event related to stacks deployed through mlstacks. I don't want to reuse that event. @bcdurak may be able to give me a better alternative.

Reply:
Okay, I named the event `DEPLOY_FULL_STACK`. On the server side, there is the event `REGISTERED_STACK` that we already had, but it will have new metadata fields:

* `wizard = True` if the full stack was registered via the CLI wizard
* `deployment = cloud-formation` if the full stack was deployed from scratch in AWS
* `provider = aws` if either happened